cloud_foundry_api/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,76 @@
+name: Cloud Foundry API
+files:
+- name: cloud_foundry_api.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - template: instances/default
+    - name: api_url
+      description: URL of the Cloud Controller API.
+      required: true
+      value:
+        type: string
+    - name: uaa_url
+      description: URL of the UAA server.
+      required: true
+      value:
+        type: string
+    - name: tls_verify
+      value:
+        example: true
+        type: boolean
+      description: Instructs the check to validate the TLS certificate of services.
```

Comment:
Let's remove this and use the http.yaml template

Reply:
Didn't want to add all of the options of that template though. Anyway to filter the ones I want to document ?

Reply:
There is a `hidden: true` that can be added for each option. Any particular reason why not document the other http options?

Reply:
Well mostly all the different authentication options are not applicable in this case